wled00/wled_server.cpp

```diff
@@ -15,6 +15,22 @@ bool isIp(String str) {
   return true;
 }
 
+void handleUpload(AsyncWebServerRequest *request, const String& filename, size_t index, uint8_t *data, size_t len, bool final){
+  if(!index){
+    request->_tempFile = WLED_FS.open(filename, "w");
+    DEBUG_PRINT("Uploading ");
+    DEBUG_PRINTLN(filename);
```

Comment:
Wouldn't it also allow re-writing the HTML pages that are being served?  While I don't want to sound alarmist, I don't think this should be allowed (at least for non-debug builds).

Reply:
Internal files are stored in flash memory and cannot be overwritten.